stable/wordpress/templates/NOTES.txt

```diff
@@ -4,7 +4,9 @@
 
   You should be able to access your new WordPress installation through
 
-  http://{{- .Values.ingress.hostname }}/admin
+  {{- range .Values.ingress.hosts }}
+  http://{{ .name }}/admin
```

Comment:
It could be https instead of http if TLS is enabled. Could we handle that case as well?

Reply:
Here's an example:

https://github.com/kubernetes/charts/blob/master/incubator/keycloak/templates/NOTES.txt#L12-L14

Reply:
Yeah, thanks for the suggestion. I added that also.